README.md

```diff
@@ -97,6 +99,33 @@ the parent. This is quite useful for launching the same command with different
 env variables or when the environment variables are too long to have everything
 in one line.
 
+On Windows you will still need to wrap environment variables passed to child
+scripts in `%` characters:
+
+```javascript
+{
+  "scripts": {
+    "parentScript": "cross-env GREET=\"Joe\" npm run childScript",
+
+    // Prints "Hello Joe" on Windows
+    "childScript": "echo Hello %GREET%"
+  }
+}
+```
+
+They will be available to subprocesses as normal however:
+
+```javascript
```

Comment:
Maybe we could get rid of the two other examples and just use this one. Nobody wants the behavior of the other two examples so they are just noise in the README.

Reply:
FWIW I have scripts that use the behaviour of the other two examples in a multi-platform Jenkins pipeline so they would have been useful to me.

Reply:
We use cross-env to declare env vars across different platforms in a common way high in the calling chain, then specialise as late as possible for paths, commands, switches, error handling and other cross platform incompatibilities.

Reply:
I think a better example would be (I haven't tested it, away from work computer):

```
"parentScript": "cross-env GREET=\"Joe\" npm run childScript",
"childScript": "cross-env echo Hello $GREET"
```

Basically everywhere you declare or use an env variable, you need to parse that command through `cross-env` or `cross-env-shell`, but it's possible to make a full scripts suite multi-platform.

Reply:
I see... Hmmm... I think that's a bit of an edge case so perhaps we could put that in [the EXAMPLES.md](https://github.com/kentcdodds/cross-env/blob/master/other/EXAMPLES.md).

Reply:
I like what @DanReyLop says :+1:

Reply:
For me:

```
"parentScript": "cross-env GREET=\"Joe\" npm run childScript",
"childScript": "cross-env echo Hello $GREET"
```

run in cmd.exe on Windows results in:

```
C:\Users\Me\temp>npm run parentScript

> temp@1.0.0 parentScript C:\Users\Me\temp
> cross-env GREET="Joe" npm run childScript


> temp@1.0.0 childScript C:\Users\Me\temp
> cross-env echo Hello $GREET

Hello %GREET%
```

Reply:
Huh, weird. I'm honestly not too concerned about this because the use case I'm interested in doesn't involve setting env variables in one script and using them in another 🤷‍♂️

Reply:
My bad, it needs to be:

```
"parentScript": "cross-env GREET=\"Joe\" npm run childScript",
"childScript": "cross-env-shell echo Hello $GREET"
```

I just use `cross-env-shell` always so I forget the differences between both commands haha

Reply:
Actually that might be the update the README needs!

Reply:
Yes, let's do it :+1:

Though I _think_ that it should be:

```
"parentScript": "cross-env GREET=\"Joe\" npm run childScript",
"childScript": "cross-env-shell \"echo Hello $GREET\""
```